benchmark/generate_diff.php

```diff
@@ -60,6 +61,24 @@ function formatDiff(?int $baseInstructions, int $headInstructions): string {
     return sprintf('%.2f%%', $instructionDiff / $baseInstructions * 100);
 }
 
+function find_benchmarked_commit_hash(string $commitHash): ?string {
+    $repeat = 10;
+
+    while (true) {
+        if ($repeat-- <= 0) {
+            fwrite(STDERR, "Count not find benchmarked commit hash\n");
+            exit(1);
+        }
+        $summaryFile = __DIR__ . '/repos/data/' . substr($commitHash, 0, 2) . '/' . $commitHash . '/summary.json';
+        if (file_exists($summaryFile)) {
+            break;
+        }
+        $commitHash = trim(runCommand(['git', 'rev-parse', $commitHash . '^'], dirname(__DIR__))->stdout);
```

Comment:
This must not print anything to the console - https://github.com/php/php-src/actions/runs/14118010354

Reply:
Good catch! I'll fix this later tonight.